<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>364</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>795</width>
    <height>364</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>795</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Esame &quot;Laboratorio di Programmazione&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="currentUploadingProgress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>741</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="totalNumberUploadedProgress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>741</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>190</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current progress</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>190</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewFileButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select new file(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadFileButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload files</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
